typescript/ts-converter/src/AstConverter.ts

```diff
@@ -309,6 +328,92 @@ export class AstConverter {
     }
 
 
+    createIdentifierExpression(name: string): Expression {
+        return this.astFactory.createIdentifierExpressionDeclarationAsExpression(
+            this.astFactory.createIdentifierDeclaration(name)
+        );
+    }
+
+    createBinaryExpression(left: Expression, operator: string, right: Expression): Expression {
+        return this.astFactory.createBinaryExpressionDeclarationAsExpression(left, operator, right);
+    }
+
+    createNumericLiteralExpression(value: string): Expression {
+        return this.astFactory.createNumericLiteralDeclarationAsExpression(value);
+    }
+
+    createBigIntLiteralExpression(value: string): Expression {
+        return this.astFactory.createBigIntLiteralDeclarationAsExpression(value);
+    }
+
+    createStringLiteralExpression(value: string): Expression {
+        return this.astFactory.createStringLiteralDeclarationAsExpression(value);
+    }
+
+    createRegExLiteralExpression(value: string): Expression {
+        return this.astFactory.createRegExLiteralDeclarationAsExpression(value);
+    }
+
+    convertIdentifierExpression(expression: ts.Expression): Expression {
+        /*console.log('Identifier:');
```

Comment:
What's the purpose of this comments?

Reply:
I forgot to remove it, it's still there from testing. I'll remove it now.